### PR TITLE
cloned Alexey's master branch and added some lines

### PR DESCRIPTION
**Pull request by kosukesato**
# General (compared to previous pull-request)
unnecessary files (.d or .ent files, etc....) were not uploaded
## run_gms.py
- renamed libra_gamess_int_path to **libra_x_path**
- added lines to use defaults.py.
- deleted parameters already defined in defaults.py.
- defined **Thermostat** object explicitly.

## main.py
- "Nstart" < 0 was deleted to initialize nuclear velocities when we want to do **NVT-MD only (without electron dynamics)**.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -296,7 +295,7 @@ def main(params):
     el = []
 
     Ttemp = 0.0
-    if params["Nstart"] < 0 and params["MD_type"] == 1: # start NA-MD interacting with thermostat @ t=0
```
Comment (alexvakimov) [inline src/main.py]: why was this removal necessary? are you sure other users don't need it?

Comment (kosukesato) [inline src/main.py]: This statement controls if nuclear velocities are initialized following Boltzmann distribution.
Since "Nstart" is the first timing for solving TD-SE, "if params["Nstart"] < 0" allows this initialization only when we do NA-MD.  If we delete this condition, we will be able to initialize the velocities even when we do NVT-MD without solving TD-SE.
Is this right?
